video/clip.go

```diff
@@ -10,9 +10,9 @@ import (
 
 type ClipStrategy struct {
 	Enabled    bool
-	StartTime  float64 `json:"start_time,omitempty"`
-	EndTime    float64 `json:"end_time,omitempty"`
-	PlaybackID string  `json:"playback_id,omitempty"` // playback-id of asset to clip
+	StartTime  int64  `json:"start_time,omitempty"`
```

Comment:
We need to make this change as well in go-api-client too, in the ClipTaskParams definition - will do